common/djangoapps/third_party_auth/models.py

```diff
@@ -204,6 +212,13 @@ def get_authentication_backend(self):
         """Gets associated Django settings.AUTHENTICATION_BACKEND string."""
         return '{}.{}'.format(self.backend_class.__module__, self.backend_class.__name__)
 
+    def available_for_login(self):
```

Comment:
Maybe this should be called `display_for_login` ? Then it's more clearly different from `accepts_logins` and that name better matches the docstring.